cli/commands/preview/tests/update.test.ts

```diff
@@ -238,4 +243,23 @@ describe( 'Preview Update Command', () => {
 		expect( mockLogger.reportError ).toHaveBeenCalledWith( expect.any( LoggerError ) );
 		expect( createArchive ).not.toHaveBeenCalled();
 	} );
+
+	it( 'should throw error if folder does not match original site and no overwrite flag', async () => {
+		const { runCommand } = await import( '../update' );
+		jest
+			.spyOn( await import( 'cli/lib/appdata' ), 'getSiteByFolder' )
+			.mockResolvedValueOnce( { id: 'different-id', path: '/other/path', name: 'Other Site' } );
```

Comment:
```suggestion
		( getSiteByFolder as jest.Mock ).mockResolvedValue( {
			id: 'different-id',
			path: '/other/path',
			name: 'Other Site',
		} );
```
If we import `getSiteByFolder` at the top, we can stick with this more familiar pattern.